tests/e2e-playwright/specs/admin/teacher/course-creation.spec.ts

```diff
@@ -50,17 +49,25 @@ describe( 'Create Courses', () => {
 		// Click "Start with default layout" button.
 		await wizardModal.startWithDefaultLayoutButton.click();
 
+		const courseOutline = coursesPage.courseOutlineBlock;
```

Comment:
Can we wrap lines 52 until 60 in a method like:
"addLesson('Lesson 1 in Module 1)"

Reply:
The method introduced here: https://github.com/Automattic/sensei/pull/6289/commits/e015b6cf0450248e0b8a207c81e15d4ea0a8e42a

Reply:
👍